src/ui/templates/icons/voiceSearchIcon.hbs

```diff
@@ -1,17 +1,17 @@
-<div class="yxt-SearchBar-voiceIconWrapper Icon--lg">
-  <div class="yxt-SearchBar-micIconWrapper {{#if customMicIconUrl}}yxt-SearchBar-CustomMicIcon{{/if}}">
+<span class="yxt-SearchBar-voiceIconWrapper Icon--lg">
```

Comment:
the only css I see is for micIconWrapper which is just `position: absolute`

Reply:
We should probably add the Icon class where it's missing just to be sure. Previously this would have been block display and making it a span would change it to inline. Inline-block is probably safest.

Reply:
i am not super familiar with handlebars js, so my understanding may be incorrect, but for voiceSearchIcon and SearchBarIcon, the children in the spans seems to be either iconPartial or builtinIcon, which use the Icon class. would this cover the display styling to be inline-block or would we need to set this explicitly again somehow? if it is the former, then i think the only instance of not using the Icon class is on line 10 of `searchBarIcon.hbs`, in which case should i just add an Icon class span wrapped right on the circle element?

Reply:
discussed offline and concluded that this PR will just address the icon classes directly and any other elements like voiceSearchIcon can be addressed separately in the future.